tests/test_calculator.py

```diff
@@ -7,4 +7,9 @@ def test_sum():
 def test_subtract():
     assert subtract(3, 3) == 0
     assert subtract(6, 4) == 2
-    assert subtract(5, 10) == -5
\ No newline at end of file
+    assert subtract(5, 10) == -5
+
+def test_divide():
+    assert divide(9, 3) == 3
+    assert divide(24, 6) == 4
+    assert divide(-10, 5) == -2
```

Comment:
Test divide by zero:

assert divide(10, 0) == -2

```suggestion
    assert divide(10, 0) == -2
```

Reply:
Done ✔️